### PR TITLE
Feat: Forced Whisper Leaks

### PR DESCRIPTION
**Pull request by Golbolco**
Courtesy of Vastined. Whispers now have a checkbox if a player wants to force their leaks. Also made slight adjustments to Lazy Cop and Strongman.

### CODE REVIEW

File: Games/types/Mafia/roles/cards/MakeKillStronger.js [hunk 1]
```diff
@@ -14,18 +14,18 @@ module.exports = class MakeKillStronger extends Card {
           return !this.data.hasUsedStrength;
         },
         action: {
-          labels: ["absolute"],
+          labels: ["strong"],
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/MakeKillStronger.js:17]: there should be an accompanying use of the "strong" label

**Comment by pikulet** [comment]
@Golbolco is there no other place in the code that uses "strong"?

File: Games/types/Mafia/roles/cards/MakeKillStronger.js (hunk 1 above)
Comment (Golbolco) [reply Games/types/Mafia/roles/cards/MakeKillStronger.js:17]: added

**Comment by Golbolco** [comment]
> @Golbolco is there no other place in the code that uses "strong"?

None. this is the only file in the codebase, I have searched a few times now

**Review by pikulet — APPROVED** [review]

File: Games/types/Mafia/roles/cards/MakeKillStronger.js [hunk 2]
```diff
@@ -14,18 +14,18 @@ module.exports = class MakeKillStronger extends Card {
...
 
             for (let action of this.game.actions[0]) {
               if (
                 action.actors.includes(this.actor) &&
-                action.hasLabels(["kill"])
+                action.hasLabel("kill")
               ) {
                 action.power = Infinity;
-                action.labels.push("absolute");
+                action.labels = [...action.labels, "absolute", "strong"];
```
Comment (DrSharky) [inline Games/types/Mafia/roles/cards/MakeKillStronger.js:28]: Should it be using absolute here? If we're using strong, it shouldn't also have absolute, right?

**Pull request merged**